### DISCUSSION TITLE
Subtitle Edit 4.0.8

### DISCUSSION THREAD

**Discussion by WillColt** [SubtitleEdit/subtitleedit#8806] [Q&A]
Hello, it's my first post in Github, so, if this isn't the right place, I apologize. The last update 4.0.8 simply broke the automatic translation into Brazilian Portuguese. Many words no longer translate and when they do, they are wrong.

**Comment by niksedk** [comment]
The change is here: https://github.com/SubtitleEdit/subtitleedit/commit/8a40c195e37ae0bd7a9bed953c82200c2f54e675

The `PORTUGUESE (BRAZIL)` is the same as the old `PORTUGUESE` from SE 4.0.7 and prior, so use that for the moment...

**Reply by WillColt** [reply]
Thanks a lot.
